cypress/e2e/signIn.cy.js

```diff
@@ -0,0 +1,50 @@
+/// <reference types="cypress" />
+describe('User should be able', () => {
+        
+    before (() => {
```

Comment:
By doing this you don't need to use cy.reload
```suggestion
    beforeEach (() => {
```